ckanext/harvest/templates/source/job/list.html

```diff
@@ -39,14 +39,14 @@ <h3 class="dataset-heading">
             <ul class="dataset-resources unstyled">
               {% if 'errored' in job.stats and job.stats['errored'] > 0 %}
                 <li>
-                  <span class="label label-important" data-diff="error">
+                  <span class="badge bg-important" data-diff="error">
```

Comment:
```suggestion
                  <span class="badge bg-danger" data-diff="error">
```
The class `label-important` has been marked as deprecated and replaced by `label-danger` which is now `bg-danger`.